packages/splitter/README.md

```diff
@@ -24,26 +24,32 @@ The `useSplitter` hook manages positioning and required
 accessibility attributes for the window splitting separator.
 
 ```jsx static
-import { useSplitter, SplitterType, SplitterOrientation } from '@zendeskgarden/container-splitter';
+import {
+  useSplitter,
+  SplitterType,
+  SplitterOrientation,
+  SplitterPlacement
+} from '@zendeskgarden/container-splitter';
 
 const Splitter = () => {
-  const { getSeparatorProps } = useSplitter({
+  const { getSeparatorProps, getPrimaryPaneProps, valueNow } = useSplitter({
     label: 'primary-pane',
-    primaryPaneId: 'primary-pane',
     type: SplitterType.VARIABLE,
     orientation: SplitterOrientation.VERTICAL,
     min: 50,
-    max: 100
+    max: 100,
+    position: SplitterPosition.TRAILS
```

Comment:
Should `SplitterPosition` be included in the `import` statement at the top? Also, would a JavaScript (non-TypeScript) project able to use these exports since they are TypeScript `enums`?

Reply:
Here is an excerpt from `index.cjs.js`:

```js
var SplitterPosition;
(function (SplitterPosition) {
  SplitterPosition["LEADS"] = "leads";
  SplitterPosition["TRAILS"] = "trails";
})(SplitterPosition || (SplitterPosition = {}));
```

So indeed TS transpiles enums to POJOs. 👍 However the transpiled files do not include them as exports:

```js
exports.SplitterContainer = SplitterContainer;
exports.useSplitter = useSplitter;
```
So I need re-export them as `export SeparatorPosition` in addition to `export declare enum SeparatorPosition`. 

I will add the missing `import` statements to the README.